tmt/steps/provision/podman.py

```diff
@@ -95,12 +102,10 @@ def start(self) -> None:
         except tmt.utils.RunError:
             needs_pull = True
 
-        # Pull image if not available or pull forced
+        # Retry pulling the image in case of network issues
+        # Temporary solution until configurable in podman itself
         if needs_pull or self.force_pull:
-            self.podman(
-                Command('pull', '-q', self.image),
-                message=f"Pull image '{self.image}'."
-                )
+            retry(self.pull_image, 5, 5, "Pulling image failed, trying again in 5 sec")
```

Comment:
Both fives should be configurable, via two new keys/options accepted by the plugin.

Reply:
Basically it would end up like these two `artemis` options, https://github.com/teemtee/tmt/blob/main/tmt/steps/provision/artemis.py#L192C2-L192C2 and https://github.com/teemtee/tmt/blob/main/tmt/steps/provision/artemis.py#L201. One for interval, one for number of pull attempts.

Reply:
It's probably going to be more than 25 seconds if we take the time `podman pull` takes to fail into account. We'll see :)